feat(vector): :sparkles: add envFrom support (#246)
vectordotdev/helm-charts: charts/vector/README.md

| dnsConfig | object | `{}` | Specify DNS configuration options for Vector Pods # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config |
| dnsPolicy | string | `"ClusterFirst"` | Specify DNS policy for Vector Pods # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy |
| env | list | `[]` | Set environment variables in Vector containers # The examples below leverage examples from secrets.generic and assume no name overrides with a Release name of "vector" |

> | envFrom | list | `[]` | Define all Secret's data as container environment variables |

Reviewer: ```suggestion
| envFrom | list | `[]` | Define environment variables from Secrets or ConfigMaps |
```
